internal/mode/static/state/dataplane/configuration.go

```diff
@@ -132,6 +174,20 @@ func newBackendGroup(refs []graph.BackendRef, sourceNsName types.NamespacedName,
 	}
 }
 
+func convertBackendTLS(btp *v1alpha2.BackendTLSPolicy) *VerifyTLS {
+	if btp == nil {
+		return nil
+	}
+	verify := &VerifyTLS{}
+	if btp.Spec.TLS.CACertRefs != nil && len(btp.Spec.TLS.CACertRefs) > 0 {
```

Comment:
similarly to the previous comment, what if we move this logic into the graph package so that we don't need to have these checks here - we can just take the parsed CA.